blocks/footer-block/features/footer/default.jsx

```diff
@@ -17,6 +17,9 @@ import {
 import { PrimaryFont } from "@wpmedia/shared-styles";
 import "./footer.scss";
 
+// switch to component link when footer converting to V2
+import Link from "./_children/link";
```

Comment:
Looks like there were changes to the footer block? was this on purpose? We only want changes to the links bar block. can you revert these changes

Reply:
yes, this is on purpose to avoid the dependency on link-bar block

Reply:
I'm working on the footer block and will fix this. it's ok for you to revert this code.

Reply:
@JackHowa : Will this affect the links appearing with `font-family: Abel`?
In Story book I'm seeing the link in a Serif font, not `$font-family-primary` (Inter) like the design in Figma.
Thanks!